src/main/java/be/seeseemelk/mockbukkit/help/HelpMapMock.java

```diff
@@ -0,0 +1,67 @@
+package be.seeseemelk.mockbukkit.help;
+
+import be.seeseemelk.mockbukkit.UnimplementedOperationException;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.help.HelpMap;
+import org.bukkit.help.HelpTopic;
+import org.bukkit.help.HelpTopicComparator;
+import org.bukkit.help.HelpTopicFactory;
+
+import java.util.*;
```

Comment:
Please do not use wildcard imports, there is absolutely no need to import the entire java.util library.
It serves no benefit here other than polluting this namespace.